File: app/src/main/java/com/sagar/watchnext/network/models/people/tvcredits/Cast.java

package com.sagar.watchnext.network.models.people.tvcredits;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Cast {

    @SerializedName("credit_id")
    private String creditId;

    @SerializedName("original_name")
    private String originalName;

    @SerializedName("id")
    private int id;

    @SerializedName("genre_ids")
    private List<Integer> genreIds ;

    @SerializedName("character")
    private String character;

    @SerializedName("name")
    private String name;

    @SerializedName("poster_path")
    private String posterPath;

    @SerializedName("vote_count")
    private int voteCount;

    @SerializedName("vote_average")
    private float voteAverage;

    @SerializedName("popularity")
    private float popularity;

    @SerializedName("episode_count")
    private int episodeCount;

    @SerializedName("original_language")
    private String originalLanguage;

    @SerializedName("first_air_date")
    private String firstAirDate;

    @SerializedName("backdrop_path")
    private String backdropPath;

    @SerializedName("overview")
    private String overview;

    @SerializedName("origin_country")
    private List<String> originCountry ;

    public String getCreditId() {
        return creditId;
    }

    public String getOriginalName() {
        return originalName;
    }

    public int getId() {
        return id;
    }

    public List<Integer> getGenreIds() {
        return genreIds;
    }

    public String getCharacter() {
        return character;
    }

    public String getName() {
        return name;
    }

    public String getPosterPath() {
        return posterPath;
    }

    public int getVoteCount() {
        return voteCount;
    }

    public float getVoteAverage() {
        return voteAverage;
    }

    public float getPopularity() {
        return popularity;
    }

    public int getEpisodeCount() {
        return episodeCount;
    }

    public String getOriginalLanguage() {
        return originalLanguage;
    }

    public String getFirstAirDate() {
        return firstAirDate;
    }

    public String getBackdropPath() {
        return backdropPath;
    }

    public String getOverview() {
        return overview;
    }

    public List<String> getOriginCountry() {
        return originCountry;
    }
}

File: app/src/main/java/com/sagar/watchnext/network/models/people/tvcredits/Cast.kt
package com.sagar.watchnext.network.models.people.tvcredits

import com.google.gson.annotations.SerializedName

class Cast {

    @SerializedName("credit_id")
    val creditId: String? = null

    @SerializedName("original_name")
    val originalName: String? = null

    @SerializedName("id")
    val id: Int = 0

    @SerializedName("genre_ids")
    val genreIds: List<Int>? = null

    @SerializedName("character")
    val character: String? = null

    @SerializedName("name")
    val name: String? = null

    @SerializedName("poster_path")
    val posterPath: String? = null

    @SerializedName("vote_count")
    val voteCount: Int = 0

    @SerializedName("vote_average")
    val voteAverage: Float = 0.toFloat()

    @SerializedName("popularity")
    val popularity: Float = 0.toFloat()

    @SerializedName("episode_count")
    val episodeCount: Int = 0

    @SerializedName("original_language")
    val originalLanguage: String? = null

    @SerializedName("first_air_date")
    val firstAirDate: String? = null

    @SerializedName("backdrop_path")
    val backdropPath: String? = null

    @SerializedName("overview")
    val overview: String? = null

    @SerializedName("origin_country")
    val originCountry: List<String>? = null
}
